lib/link.js

```diff
@@ -41,6 +42,10 @@ export default class Link extends Component {
   }
 
   render () {
+    if (Children.count(this.props.children) > 1) {
+      warn(`Warning: 'Link' accepts only one child element.`)
+    }
```

Comment:
@timneutkens `React.PropTypes.element.isRequired` is what we want ? https://facebook.github.io/react/docs/typechecking-with-proptypes.html#requiring-single-children